lib/mac-learning.c

```diff
@@ -30,6 +30,10 @@
 #include "util.h"
 #include "vlan-bitmap.h"
 
+#if defined(P4OVS)
+#include "openvswitch/ovs-p4rt.h"
+#endif //P4OVS
```

Comment:
It's unusual to have a comment on an #endif when the conditional block is this short.

Reply:
Removed